src/cart/crt_hg.c

```diff
@@ -1284,6 +1284,17 @@ crt_hg_reply_send(struct crt_rpc_priv *rpc_priv)
 
 	D_ASSERT(rpc_priv != NULL);
 
+	if (D_LOG_ENABLED(DB_NET)) {
+		uint64_t hlc = crt_hlc_get();
+
+		if (hlc > rpc_priv->crp_create_hlc) {
+			uint64_t delay = crt_hlc2msec(hlc - rpc_priv->crp_create_hlc);
```

Comment:
ditto

Reply:
This is very interesting to see how long time passed for reply prepare. Again, rpc_priv->crp_create_hlc will be initialized when RPC income and this message show how long we are process it and prepare reply to send back. If this time is more than CRT_RPC_TIMEOUT this reply will be dropped by remote site because of original RPC is expired by timeout.

Reply:
ok, thanks for explanations.